int32 id
float64 tag_size
geometry_msgs/Point[] corners2d
geometry_msgs/PoseStamped pose